packages/driver/src/httpScram.ts

```diff
@@ -24,6 +24,31 @@ function b64ToUtf8(str: string): string {
   return utf8Decoder.decode(decodeB64(str));
 }
 
+/**
+ * Parses SCRAM-SHA-256 authentication parameters from a string.
+ * @param authParamsString a string of comma-separated key-value pairs
+ * @returns an object with `sid` and `data` properties, decoded string or `null` when not present
+ */
```

Comment:
Couple of small nits:

1. No need to add comments and JSDocs to an internal function.
2. Our style is to put these internal functions after the exported functions since they play more of a supporting/abstracting role.